src/index.jsx

```diff
@@ -54,6 +61,29 @@ class CircularColor extends PureComponent {
     });
   }
 
+  handleClick = (event) => {
+    const { size } = this.props;
+    const { innerRadius, outerRadius } = this.statics;
+
+    const { x: xBlock, y: yBlock } = event.currentTarget.getBoundingClientRect();
+
+    const x = event.clientX - xBlock - size / 2;
+    const y = event.clientY - yBlock - size / 2;
+
+    const angle = -Math.atan2(y, x);
+
+    const outX = Math.abs(Math.floor((outerRadius) * Math.cos(angle)));
+    const outY = Math.abs(Math.floor((outerRadius) * Math.sin(angle)));
+
+    const inX = Math.abs(Math.floor((innerRadius) * Math.cos(angle)));
+    const inY = Math.abs(Math.floor((innerRadius) * Math.sin(angle)));
+
+    if (Math.abs(x) >= inX && Math.abs(x) <= outX
```

Comment:
I'm not sure about this. I'd recommend you should add something like "epsilon" for better "look and feel" on mobile devices

Reply:
Should this "epsilon" be inside the props of the component CircularColor? If yes, what the default value?